lib/assertions.js

```diff
@@ -55,33 +55,35 @@ const identities = {
   ],
 }
 
-const makeCorpPass = ({ NRIC, UEN }) => render(
+const corpPassNric = process.env.MOCKPASS_NRIC || identities.corpPass[0].NRIC
+const uen = process.env.MOCKPASS_UEN || identities.corpPass[0].UEN
+
+const makeCorpPass = (source = { NRIC: corpPassNric, UEN: uen }) => render(
   TEMPLATE,
   {
     issueInstant: moment.utc().format(),
-    name: UEN,
-    value: base64.encode(render(corpPassTemplate, { NRIC, UEN })),
+    name: source.UEN,
+    value: base64.encode(render(corpPassTemplate, source)),
   }
 )
 
-const makeSingPass = (NRIC) => render(TEMPLATE,
+const nric = process.env.MOCKPASS_NRIC || identities.singPass[0]
+const recipient = process.env.SERVICE_PROVIDER_RECIPIENT || "http://localhost:8080/saml/SSO"
```

Comment:
```suggestion
const recipient = process.env.SERVICE_PROVIDER_RECIPIENT || "http://sp.example.com/demo1/index.php?acs"
```
I would prefer to have the silly default found in the original template if the env var is not provided at runtime

Reply:
I've used the old default value :)